src/screens/ProfileScreen.js

```diff
@@ -18,6 +18,7 @@
 
 import React from 'react';
 import Icon from '@expo/vector-icons/FontAwesome';
+import Resource from '../components/Resource';
```

Comment:
nit: could you put this in the other block of imports - we typically keep imports from within the app seperated from imports from outside